x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/results/AnomalyScoreExplanation.java

```diff
@@ -78,6 +81,7 @@ public AnomalyScoreExplanation(StreamInput in) throws IOException {
         this.upperConfidenceBound = in.readOptionalDouble();
         this.highVariancePenalty = in.readOptionalBoolean();
         this.incompleteBucketPenalty = in.readOptionalBoolean();
+        this.multimodalDistribution = in.readOptionalBoolean();
```

Comment:
Should this be guarded with a version check? I.e.:

```
if (in.getVersion().onOrAfter(Version.V_8_7_0) {
   ...
}
```

Reply:
Good spot. Fixed it in 562e992